src_rebuild/GAME/C/MAIN.C

```diff
@@ -2637,6 +2637,63 @@ int redriver2_main(int argc, char** argv)
 			i++;
 		}
 #endif
+		else if (!_stricmp(argv[i], "-dump_missions"))
+		{
+			char input[32] = { NULL };
+
+			// load from BLK
+			sprintf(input, "MISSIONS\\MISSIONS.BLK");
+
+			if (FileExists(input))
```

Comment:
it needs better to be in separate function... or we need it in OpenDriver2Tools

Reply:
Not much code is required to do this. I would only move it to OpenDriver2Tools if you planned on making a (de)compiler tool.